public/locales/ru/common.json

```diff
@@ -26,7 +26,6 @@
   "products": "Продукты",
   "services": "Услуги",
   "others": "Другие",
-  "volunteer": "Доброволец",
```

Comment:
keep with the respective translation for Volunteering
and check signup.volunteering.header to be the same